Fix "Class 'I18n' not found" error

lukasbestle opened the pull request:
## Describe the PR

Fixes errors that occur because of the wrong locale set for `LC_CTYPE` (introduced in 3.2.5-rc.1).

Also two other minor improvements to the code:

- Also pass language code as locale fallback for the new `Api` locale feature
- Add missing `use` declaration in the file that caused the error originally

## Related issues

- Fixes #2128

## Todos

- [x] Add unit tests for fixed bug/feature
- [x] Pass all unit tests
- [x] Fix code style issues with CS fixer and `composer fix`
- [x] If needed, in-code documentation (DocBlocks etc.)

lukasbestle committed:
Only set specific locales in Api class #2128

lukasbestle committed:
Also pass language code as locale fallback #2128

lukasbestle committed:
Add missing use declaration #2128

afbora commented on config/sections/mixins/empty.php:
```diff
@@ -1,5 +1,7 @@
 <?php
 
+use Kirby\Toolkit\I18n;
```
Now it works without this line because of other updates. Main issue was about `LC_CTYPE`

lukasbestle replied:
Awesome! Thanks for verifying. 👍

lukasbestle committed:
Fix remaining use declarations in config dir #2128

Those files don’t have namespaces, so we didn’t notice that we forgot to import some classes (because the aliases kicked in).

afbora committed:
Add missing use declaration

afbora committed:
Add missing use declaration

lukasbestle committed:
Merge pull request #2132 from afbora/fix-2128-missing-use-declarations

Add missing use declarations
